src/pal/tests/palsuite/exception_handling/pal_sxs/test1/dlltest1.cpp

```diff
@@ -30,12 +39,14 @@ int DllTest1()
 
     PAL_TRY(VOID*, unused, NULL)
     {
+        EHGuard(NULL);
```

Comment:
Understood. I misunderstood the intent of the original change and now it is clear.